### ISSUE TITLE
Missing :endif error on launch

### ISSUE THREAD

**Issue by eater** [reedes/vim-textobj-sentence#6]
I've just installed this on my Vim 7.4 and when I open a file in Vim, I get these messages:

    Error detected while processing function textobj#sentence#init..textobj#user#plugin..369:
    line   15:
    E605: Exception not caught: Unknown command: 'pattern'
    Error detected while processing function textobj#sentence#init..textobj#user#plugin:
    line   15:
    E171: Missing :endif                              

Not sure how best to track this one down.

**Comment by reedes** [comment]
That I haven't seen before.

Are you using a recent version of the https://github.com/kana/vim-textobj-user plugin? That's a key dependency.

**Comment by eater** [comment]
Yes, it's the latest version. I deleted and reinstalled all the plugins just to be sure. I'm running vim-textobj-quote as well, and it doesn't give me any trouble.

**Comment by reedes** [comment]
There is a 'pattern' property in vim-textobj-user config for my sentence plugin, but I'm not able to see how that is being mis-executed command where the error is being thrown.

How are you initializing the sentence plugin in your .vimrc?

**Comment by eater** [comment]
I use Vundle, so everything's sourced at the top of the vimrc.

**Comment by reedes** [comment]
Are you initializing by file type, as in:

```vim
set nocompatible
filetype plugin on

augroup textobj_sentence
  autocmd!
  autocmd FileType markdown call textobj#sentence#init()
augroup END
```

**Comment by eater** [comment]
Yes, I forgot that that's in there:

    augroup textobj_quote
      autocmd!
      autocmd FileType markdown call textobj#quote#init()
      autocmd FileType text call textobj#quote#init({'educate': 0})
    augroup END

    augroup textobj_sentence
      autocmd!
      autocmd FileType markdown call textobj#sentence#init()
      autocmd FileType text call textobj#sentence#init()
    augroup END

**Comment by reedes** [comment]
Hmm, pasting those two groups in didn't fail for me, unfortunately.

Doe the error still happen if you reduce to a single block?

```vim
augroup textobj_quote
  autocmd!
  autocmd FileType markdown call textobj#quote#init()
                                             \ | call textobj#sentence#init()
  autocmd FileType text call textobj#quote#init({'educate': 0})
                                  \ | call textobj#sentence#init()
augroup END
```

**Comment by eater** [comment]
Yes, the error still happens with that block instead. And I don't have any other "autocmd FileType" lines elsewhere.

**Comment by reedes** [comment]
Any chance you could post your vimrc via gist or as a public project (such as I do here: https://github.com/reedes/vim-config) ?

I'll then try to reproduce the issue locally. Hopefully it'll fail where I can diagnose the cause.
